src/aiovodafone/api.py

```diff
@@ -584,7 +591,7 @@ def convert_uptime(self, uptime: str) -> datetime:
             minutes=m,
         )
 
-    async def login(self) -> bool:
+    async def login(self, force_logout: bool = False) -> bool:  # noqa: ARG002
```

Comment:
```suggestion
    async def login(self, force_logout: bool = False) -> bool: 
```

Reply:
Yeah no the pipeline will just fail if we do this.

Reply:
Alternative: do `_ = force_logout` but that's just another hack to trick the linter.